DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -179,11 +196,15 @@ private void GameBroadcastTimer_TimeElapsed(object sender, EventArgs e)
             {
                 RandomSeed = new Random().Next();
                 RefreshMapSelectionUI();
+                btnChangeTunnel.Enabled = true;
+                btnChangeTunnel.Visible = true;
```

Comment:
Use `btnChangeTunnel.Enable()` instead of setting the `Enabled` and `Visible` attributes manually.